server/src/controllers/client_timeline_entries/addClientTimelineEntry.js

```diff
@@ -0,0 +1,96 @@
+const logger = require("pino")();
+const ClientTimelineEntry = require("../../models/client_timeline_entry.model");
+const {
+  submitPlacementUpdateEntryInTimelines,
+} = require("../../utils/placement_entry_util");
+
+const addClientTimelineEntryRequestHandler = async (req, res) => {
+  try {
+    // eslint-disable-next-line camelcase
+    const { type, title, body, client, job_lead } = req.body.entry;
+
+    const { user } = req;
+
+    const validTypes = ["update", "contact", "placement", "note"];
```

Comment:
Also placement should probably be removed from the client timeline api since you can't place a client from their timeline in the designs. Rather the entry in the client timeline should be created when the client is placed through the employer or job lead timeline.